.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+  "editor.tabSize": 2,
+  "editor.insertSpaces": true,
+  "files.trimTrailingWhitespace": true
+}
```

Comment:
I'm guessing that this file is needed in addition to the .editorconfig file to work correctly w/ VSCode?  Works correctly for me (I'm running the Editor Config extension on VSCode).